### DISCUSSION TITLE
Apple warning ITMS 909001

### DISCUSSION THREAD

**Discussion by MatheusGop** [expo/expo#16016] [Archived: Questions]
Hi, I just submitted my app to my appstoreconnect account, it returned with the following issue:

ITMS-90901: Missing full-screen support for the latest iPad mini display. The “ExpoKitApp.app” bundle includes UIRequiresFullScreen=YES in the Info.plist or supports only a subset of UISupportedInterfaceOrientations, and is built with the 14.1 SDK. To take advantage of the full screen size, recompile with Xcode 13 and the SDK for iPadOS 15 or later.

I already tried to put this on my Info.plist, but it didn't work, so what more can I do? I'm using expo managed

**Comment by anujdivesh** [comment]
Hi @MatheusGop  I seem to got it working, but not sure if this is the fix, I updated my app to latest sdk and included in the app.json
```
"ios": {
      "supportsTablet": true,
      "infoPlist": {
        "UIRequiresFullScreen": true
      },
      "bu.....
```
can you see if it works?

**Reply by edsonjuniornarvaes** [reply]
it worked for me
